docs/versionhistory.rst

```diff
@@ -25,6 +25,7 @@ This library adheres to `Semantic Versioning 2.0 <http://semver.org/>`_.
 
   - The ``TaskStatus`` class is now generic, and should be parametrized to indicate the
     type of the value passed to ``task_status.started()``
+  - The ``Listener`` class is now covariant in its stream type.
```

Comment:
```suggestion
  - The ``Listener`` class is now covariant in its stream type
```

Reply:
I force-pushed a change that removes the dot.